src/command-configs/fetchCommandsConfiguration.ts

```diff
@@ -54,7 +56,7 @@ export async function fetchCommandsConfiguration(
 }
 
 export function getDocsUrl(commitHash: string): string {
-  return new URL(path.join(config.cmdBotUrl, DOCS_PATH, getDocsFilename(commitHash))).toString()
+  return new URL(path.join(config.cmdBotUrl, DOCS_URL_PATH, getDocsFilename(commitHash))).toString()
```

Comment:
Using `path.join` in URLs is probably a mistake, as the purpose of `path.join` is to generate OS-specific slashes, while URLs always use `/`

Reply:
Although, it will work anyway, so changing it is up to you
<img width="463" alt="Screen Shot 2023-01-25 at 16 37 13" src="https://user-images.githubusercontent.com/588262/214606513-65e6c1e4-ece4-4387-b0b0-0c6658ceaa24.png">

Reply:
thanks, i'd keep it for now